memory_32_8.x

```diff
@@ -0,0 +1,9 @@
+/*
+    Set memory sizes to the smallest values found in any of the  LPC800 parts we
```

Comment:
This should be updated, since it's a specific value now (same in the 2 other memory.x files)

Reply:
Right. I've pushed another commit to remove them.